src/game/SpellEffects.cpp

```diff
@@ -2936,37 +2942,6 @@ void Spell::EffectSummonPet(SpellEffectIndex eff_idx)
         return;
     }
 
-    Pet* NewSummon = new Pet;
-
-    // petentry==0 for hunter "call pet" (current pet summoned if any)
-    if (m_caster->GetTypeId() == TYPEID_PLAYER && NewSummon->LoadPetFromDB((Player*)m_caster, petentry))
-    {
-        if (NewSummon->getPetType() == SUMMON_PET)
-        {
-            // Remove Demonic Sacrifice auras (known pet)
```

Comment:
As soon as we have worked out the kinks, the bugs and the regressions in my pr and merged it with classic, I will be working on porting all of what's relevant to TBC, and after that, all that's relevant from TBC on to Wrath (and whatever was relevant to wrath from vanilla and not tbc... like there's actually anything, didn't they completely fuck up hunters pets by wrath?)